tests/publiccloud/check_services.pm

```diff
@@ -21,37 +21,39 @@ sub run {
 
     my $instance = $args->{my_instance};
 
-    # Debug
-    $instance->ssh_script_run('systemctl --no-pager list-units');
+    my %known_failing_services = (
+        'systemd-vconsole-setup' => 'bsc#1249902 - systemd-vconsole-setup.service failed to load',
+        'NetworkManager-wait-online' => 'This service throws false positives sometimes'
+    );
 
-    # Check if there are any failed services
-    # 1. Get the list of failed services from the SUT
-    my $failed_services_output = $instance->ssh_script_output('sudo systemctl --failed --no-legend --plain | awk "{print \\$1}"');
+    my $failed_services_output = $instance->ssh_script_output(
+        'sudo systemctl --failed --no-legend --plain --no-pager | awk "{print \\$1}"'
+    );
 
-    # 2. Split the output into a Perl array of service names
     my @failed_services = split /\s+/, $failed_services_output;
 
     if (!@failed_services) {
         record_info("All SVCs good", "No failed services found.");
         return;
-    }
-
-    # 3. Loop through each service name in Perl
-    my $failed = 0;
-    foreach my $service (@failed_services) {
-        # 4. Capture the log output for the service into a Perl variable
-        my $log_output = $instance->ssh_script_output("sudo journalctl -u $service --no-pager");
-
-        # 5. Pass the captured logs as the text for record_info.
-        #    This creates a single, expandable log entry in the openQA web UI.
-        if ($service =~ /systemd-vconsole-setup/) {
-            record_soft_failure("bsc#1249902 - systemd-vconsole-setup.service failed to load:\n$log_output");
-            next;
+    } else {
+        my $failed = 0;
+        foreach my $service (@failed_services) {
+            if (exists %known_failing_services{$service}) {
+                record_soft_failure(
+                    "%known_failing_services($service)\n$log_output"
```

Comment:
```suggestion
                    "%known_failing_services($service)"
```

log_output will be collected only in case when it is something unknown ( and I think it make sense )

Reply:
Ok, I added it just in case the failure was some different issue under the same service...